### DISCUSSION TITLE
Re-build needed for storefront but not for dashboard on updating saleor

### DISCUSSION THREAD

**Discussion by CuriousLearner** [saleor/saleor#9273] [Q&A]
Hi,

I've set-up saleor projects locally on my system *without* using docker. I realized that when generate the build for dashboard, it works seamlessly and all data that is in saleor core is shown and can be updated from there.

However, when I run the (now deprecated) saleor-storefront, if I don't run it through npm run build it doesn't render the content automatically based on the data received. So, for example, if I add a few more products from dashboard, for them to be visible in storefront, I've to make a build again. Why is that a case? Am I doing something wrong?

PS: I'm mostly into BE development & new to React / Next -- so please bear with me :)

**Comment by ashishkpaul** [comment]
Dashboard and Storefront are static apps
https://github.com/saleor/saleor/discussions/8977#discussioncomment-2007680
